attributecert/attributecert.go

```diff
@@ -320,7 +320,7 @@ type TCGCredentialType struct {
 type FipsLevel struct {
 	Version string
 	Level   asn1.Enumerated
-	Plus    bool
+	Plus    bool `asn1:"optional,default=false"`
```

Comment:
Similar to the comment about Version, bool should already default to false. Is marking as optional enough?